### PR TITLE
INT-1110 create ORB

### PR DESCRIPTION
**Pull request by fernau**
### Description
Calls the groovy wrapper (#2) from the ORB. 
This is actually working and manually tested on CircleCI.
A test project is here: https://github.com/circlecitest18/circlecitest

### CODE REVIEW

File: orb.yml [hunk 1]
```diff
@@ -11,19 +11,33 @@ commands:
...
+        unzip apache-groovy-binary.zip
+        mv groovy-<< parameters.groovy-version>> groovy
 
   archive:
+    description: "Archives artifacts to Nexus Repository Manager"
+    parameters:
+      publisher-parameters:
+        type: string
+        # TODO this will be parametrized - a WIP shortcut
+        default: "--username admin --password admin123 --serverurl http://localhost:8081/ --filename /tmp/workspace/test1-1.0-SNAPSHOT.jar --format maven2 -CgroupId=com.example -CartifactId=myapp -Cversion=1.0 -Aextension=jar --repository maven-releases"
```
Comment (rjackson64840) [inline orb.yml]: These params all seem to match up with the NexusPublisher script

**Review by rjackson64840 — APPROVED** [review]

File: orb.yml [hunk 2]
```diff
@@ -11,19 +11,69 @@ commands:
...
+        --username <<parameters.username>>
+        --password <<parameters.password>>
+        --serverurl <<parameters.serverurl>>
+        --filename <<parameters.filename>>
+        --format <<parameters.format>>
+        -CgroupId=<<parameters.groupId>>
+        -CartifactId=<<parameters.artifactId>>
+        -Cversion=<<parameters.version>>
+        -Aextension=<<parameters.extension>>
+        --repository <<parameters.repository>>
```
Comment (rjackson64840) [inline orb.yml]: I'm not a yaml expert, but the '>' notation at the beginning defines a block of text where newlines become spaces and adds a newline at the very end.  See here:  https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines

Comment (fernau) [reply orb.yml]: thanks looks great

File: orb.yml [hunk 3]
```diff
@@ -11,19 +11,104 @@ commands:
...
+        description: "Version of Apache groovy to download and install"
+        type: string
+        default: "2.5.3"
     executor: default
     steps:
-    - install
+    - install:
+      groovy-version: << parameters.groovy-version >>
+    - archive:
+        username: << parameters.username >>
```
Comment (bigspotteddog) [inline orb.yml:108]: Very minor: The parameter value spacing in `<< >>` is different between this `archive` block and the `run` block above.

**Review by bigspotteddog — APPROVED** [review]
+1

File: orb.yml (hunk 3 above)
Comment (fernau) [reply orb.yml:108]: make <<>> consistent

**Pull request merged**